### DISCUSSION TITLE
feature_pysu2_fsi_adjoint CDiscAdjSinglezoneDriver iZone error

### DISCUSSION THREAD

**Discussion by yiren6** [su2code/SU2#1813] [Q&A] [answered]
**Describe the bug**
When calling pysu2 direct and pysu2ad discrete adjoint in a single Python script, there is a segfault caused by initialization of iZone.
**solution**
In feature_pysu2_fsi_adjoint/driver/CDiscAdjSinglezoneDriver::Run_Residual() {
       /*--- Initialize the linear solver iterations ---*/
// line 269
        auto nIter = min(nKrylov_Iter - 2ul, config_container[iZone]->GetnQuasiNewtonSamples() - 2ul);
}
handle iZone should be changed to ZONE_0.  iZone was not defined in this function. 

**Desktop (please complete the following information):**
OS: Ubuntu 20.04LTS
MPI implementation and version: mpich 4.0.1
SU2 Version: 7.3
Python Version: 3.9.7
SWIG Version: 4.0.2

**Comment by pcarruscag** [comment, marked as answer]
I suggest you comment directly on the corresponding PR for that branch or that you use a Discussion to get in thouch with the authors.
Issues that concern the broader community are those in the develop and master branches.

**Reply by yiren6** [reply]
Thank you!
